gprofiler/main.py

```diff
@@ -121,12 +121,16 @@ def __init__(
         # the latter can be root only. the former can not. we should do this separation so we don't expose
         # files unnecessarily.
         self._temp_storage_dir = TemporaryDirectoryWithMode(dir=TEMPORARY_STORAGE_PATH, mode=0o755)
+
```

Comment:
Can be removed

Reply:
(I meant the space :) I commented on the space line)

Reply:
Ah! Done.